From: Mike Hommey <mh@glandium.org>
Date: Mon, 21 May 2018 12:14:06 +0900
Subject: Bug 1463036 - Use HAVE_ARM_NEON instead of BUILD_ARM_NEON for
 nsUTF8UtilsNEON.cpp. r?build

There is a subtle difference between the two, and the conditions used in
nsUTF8Utils.h correspond to HAVE_ARM_NEON, not BUILD_ARM_NEON.
---
 xpcom/string/moz.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xpcom/string/moz.build b/xpcom/string/moz.build
index 65dd9c3129fd..4689b8fcb971 100644
--- a/xpcom/string/moz.build
+++ b/xpcom/string/moz.build
@@ -57,7 +57,7 @@ if CONFIG['INTEL_ARCHITECTURE']:
     SOURCES += ['nsReadableUtilsSSE2.cpp']
     SOURCES['nsReadableUtilsSSE2.cpp'].flags += CONFIG['SSE2_FLAGS']
 
-if CONFIG['BUILD_ARM_NEON'] or CONFIG['CPU_ARCH'] == 'aarch64':
+if CONFIG['HAVE_ARM_NEON'] or CONFIG['CPU_ARCH'] == 'aarch64':
     SOURCES += ['nsUTF8UtilsNEON.cpp']
     SOURCES['nsUTF8UtilsNEON.cpp'].flags += CONFIG['NEON_FLAGS']
 
